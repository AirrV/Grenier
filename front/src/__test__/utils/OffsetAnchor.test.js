import React from 'react';
import renderer from "react-test-renderer";
import OffsetAnchor from '../../components/utils/OffsetAnchor';

it("Renders the welcome page", () => {
  const tree = renderer
    .create(<OffsetAnchor />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})