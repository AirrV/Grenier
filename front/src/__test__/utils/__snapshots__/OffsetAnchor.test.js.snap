// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the welcome page 1`] = `
<div
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    style={
      Object {
        "left": 0,
        "position": "absolute",
        "top": -70,
      }
    }
  />
</div>
`;
